eZ/Publish/Core/Base/Container/Compiler/TaggedServiceIdsIterator/BackwardCompatibleIterator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Base\Container\Compiler\TaggedServiceIdsIterator;
+
+use Iterator;
+use IteratorAggregate;
+use Symfony\Component\DependencyInjection\TaggedContainerInterface;
+
+final class BackwardCompatibleIterator implements IteratorAggregate
```

Comment:
```suggestion
/**
 * @internal
 */
final class BackwardCompatibleIterator implements IteratorAggregate
```
right?

Reply:
Sure. Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/51/commits/77373994cd6e94f32c282e29d7498199bbca2fb7.